README.md

```diff
@@ -13,6 +13,7 @@ A simple finite element toolbox written in Julia.
 
 ## Documentation
 
+[![][docs-stable-img]][docs-stable-url]
 [![][docs-dev-img]][docs-dev-url]
```

Comment:
I would prefer only including the link to http://ferrite-fem.github.io/Ferrite.jl/ (which redirect to /stable).